src/style.css

```
@@ -447,6 +447,361 @@ body {
+}
+
+.skill-editor-body textarea {
+  width: 100%;
+  min-height: 240px;
+  padding: 12px;
+  background: #1a1a2e;
+  border: 1px solid #2d4a3e;
+  border-radius: 6px;
+  color: #e5e7eb;
+  font-size: 13px;
+  font-family: 'Consolas', 'Monaco', monospace;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)